src/main/java/net/atos/zac/mail/MailService.java

```diff
@@ -200,13 +206,13 @@ private EnkelvoudigInformatieObjectData createDocumentInformatieObject(final Str
     private byte[] createPdfDocument(final String verzender, final String ontvanger, final String subject,
```

Comment:
It would be nice if we could add at least one Unit Test for this method

Reply:
https://dimpact.atlassian.net/browse/PZ-1489